web/views/llm.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from django.http import JsonResponse
+from django.shortcuts import get_object_or_404
+from django.views.decorators.csrf import csrf_exempt
+
+from web.views.utils import has_permission_for_submit
+from common.models import Submit
+
+
+@csrf_exempt
+def post_submit_summary_result(request, assignment_id, login, submit_num):
+    if request.method != "POST":
+        return JsonResponse({"error": "Invalid request method."}, status=405)
+
+    submit = get_object_or_404(
+        Submit, assignment_id=assignment_id, student__username=login, submit_num=submit_num
+    )
+
+    has_permission_for_submit(request, submit)
```

Comment:
Let's use the same authentication mechanism here as we do for `upload_results` (feel free to move it into a separate method, e.g. `authenticate_evaluator_request` or something). We should not allow anyone else to authenticate here.